### PR TITLE
Toggleable overlays and changeable basemaps

### PR DESCRIPTION
**Pull request by rmartz**
## Overview

Adds a Leaflet layer control to the Map page to configure which basemap is used, as well as offers options for drawing the regions, major settlements and roadways. The Data page is unchanged and only uses the defaults (No overlays and the default basemap).
### Demo

screen shot 2016-08-31 at 3 16 08 pm
screen shot 2016-08-31 at 3 16 24 pm
screen shot 2016-08-31 at 3 16 36 pm
### Testing
- Navigate to Map page
- Changing the radio box should result in different basemaps being used under the map
- Toggling checkboxes should result in adding the corresponding information on top of the map

### CODE REVIEW

File: src/i18n/mn.js [hunk 1]
```diff
@@ -60,6 +60,10 @@ export const mn = {
   MAP_VIZ_INFR_INDOOR_SANITATION_PCT: '!% of Population with Indoor Sanitation 2010',
   MAP_VIZ_INFR_PIPED_WATER_PCT: '!% of Population with Access to Piped Water 2010',
   MAP_VIZ_INFR_SOLID_WASTE_DISPOSAL_PCT: '!% of Population with Organized Solid Waste Disposal 2010',
-  MAP_VIZ_INFR_INTERNET_ACCESS_PCT: '!% of Population with Access to Internet 2010'
-
+  MAP_VIZ_INFR_INTERNET_ACCESS_PCT: '!% of Population with Access to Internet 2010',
```
Comment (CloudNiner) [inline src/i18n/mn.js:65]: Looks like maybe some missed keys in this file?

**Comment by CloudNiner** [comment]
Looks like the regional overlay might still exist on all of the data visualizations?

**Comment by CloudNiner** [comment]
+1 really nice.

File: src/i18n/mn.js (hunk 1 above)
Comment (rmartz) [reply src/i18n/mn.js:65]: Whoops, good catch! I'll fix those up

**Comment by rmartz** [comment]
@CloudNiner Yup, the regional overlay is included in the CartoDB maps currently, it should be removed soon so the toggleable overlay can be used instead.

**Pull request merged**